stock-tracker/src/features/overview/component/Overview.tsx

```diff
@@ -7,17 +7,17 @@ import "./Overview.css";
 import { AppState } from "../../../store/rootReducer";
 
 export const Overview = () => {
-  const { companyOverview, loading, error } = useSelector(
-    (state: AppState) => state.overview
+  const { companyDetails: companyOverview, loading, error } = useSelector(
+    (state: AppState) => state.companyOverview
   );
 
   return (
     <div className="overview">
       <h1>COMPANY OVERVIEW</h1>
-      {error ? (
-        <ErrorMessage feature={"Company Overview"} />
-      ) : loading ? (
+      {loading ? (
         <Loading />
+      ) : error || !companyOverview ? (
+        <ErrorMessage feature={"Company Overview"} />
```

Comment:
q: I'm a bit confused why you're passing the message down as a prop. You shouldn't need to use the `{  }` syntax around the message, as you're just passing down a string

Reply:
nit: Consider changing the name of `feature` to something like `message`, as you're not really passing down a `feature` to that component

Reply:
That's true! I didn't know it works without {} 👍 
I call it feature because it's the same error message, the only thing that changes is the name of a feature, so it made sense to me.